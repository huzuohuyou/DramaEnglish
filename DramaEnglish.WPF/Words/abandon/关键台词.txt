Abandon ship! Abandon ship!
全体弃船 弃船!
