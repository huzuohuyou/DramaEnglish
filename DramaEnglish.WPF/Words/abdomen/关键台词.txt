Is everyone all right?!
大家都没事吧
- Abdomen! - What?!
- 腹部 - 什么
Who the hell fired?!
谁开的枪
