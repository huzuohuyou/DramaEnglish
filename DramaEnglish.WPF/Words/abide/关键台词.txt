The city council will abide this resolution
市议会将会履行这项决议
