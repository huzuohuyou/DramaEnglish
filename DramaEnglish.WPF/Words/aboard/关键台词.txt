All aboard! All aboard!
全部上船! 全部上船!
