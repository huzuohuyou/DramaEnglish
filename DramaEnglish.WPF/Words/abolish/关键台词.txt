by abolishing slavery
只是废除奴隶制
