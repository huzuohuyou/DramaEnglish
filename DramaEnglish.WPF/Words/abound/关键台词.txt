Irony abounds.
真是讽刺
