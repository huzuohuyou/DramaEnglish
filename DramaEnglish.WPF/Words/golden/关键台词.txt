Daniel Golden Daniel golden.
00:02:21,940
